src/biocommons/seqrepo/fastadir/fastadir.py

```diff
@@ -123,11 +120,12 @@ def fetch(self, seq_id, start=None, end=None):
         fabgz = self._open_for_reading(path)
         return fabgz.fetch(seq_id, start, end)
 
+    @functools.lru_cache(maxsize=os.environ.get("SEQREPO_LRU_CACHE_MAXSIZE"))
```

Comment:
`os.environ.get` with one arg will return `None` or a `str` type. The get call should be wrapped in an explicit `int()` conversion.

Reply:
Good catch, Kyle.